src/ConcordiumNetSdk/Client/ClientConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+namespace ConcordiumNetSdk;
+
+/// <summary>
+/// Specifies connection settings for the <see cref="Client"/>.
+/// </summary>
+public class ClientConfiguration
+{
+    /// <summary>
+    /// The request timeout in seconds.
+    /// </summary>
+    public ulong Timeout { get; private set; }
```

Comment:
No. This refers to the gRPC deadline which is the point in time where the procedure expires and a `DeadlineExceededError` is thrown. So this is specific for the call.

It is possible to set channel options on the client side, for instance specifying that the channel should be kept alive without calls, but I am not sure what 

`new ChannelOption("grpc.keepalive_permit_without_calls", 1)`

However I believe care should be taken, since I am not sure what happens if the server dies when this is set. I will have to test it manually. The various flags are documented here, and I will leave it here for now and address it once everything else is implemented:

https://github.com/grpc/grpc/blob/master/doc/keepalive.md

Reply:
Ok, we should clarify what the defaults are and document that.

And provide a way to set additional options.